osquery/tables/networking/darwin/wifi.mm

```diff
@@ -126,7 +126,7 @@ QueryData genKnownWifiNetworks(QueryContext& context) {
   CFTypeRef networks = CFDictionaryGetValue(plist, cfkey);
   CFRelease(cfkey);
   if (networks == nullptr) {
-    VLOG(1) << "Key not found : " << key;
+    LOG(INFO) << "Key not found : " << key;
```

Comment:
This is not helpful, key not found for what? My recommendation is to keep it a verboselog.